hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/HdfsStore.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.mesos.hdfs.state;
```

Comment:
This interface is super important IMO.   This allows us to remove ZK as a requirement by creating an impl for another store... or an in-memory store in tests.